### ISSUE TITLE
More features

### ISSUE THREAD

**Issue by peter279k** [Josantonius/PHP-DataType#2]
As title, the data type only let object convert to array.
How about other converting data type?
For example, the strings convert to arrays, integers convert to floats, XML strings convert to JSON strings and so on.
Maybe it can let the data type convert each other.
If possible, I can help you adding this features.
What do you think?

Thanks.

**Comment by Josantonius** [comment]
Exactly, that is the main idea for this class, to facilitate the conversion between different data types. 

For now it is in process, so any contribution is appreciated. Feel free to implement the features you comment on :wink: 

The conversion between XML and JSON I think would be better in [PHP-Json](https://github.com/Josantonius/PHP-Json), since it wouldn't be a data type in itself. What do you think?

Sorry for the delay!

**Comment by peter279k** [comment]
Hi @Josantonius, thank you for your reply.
I think the conversion between XML and JSON it can add this feature in ```PHP-JSON```.
And other conversion of data types can be added in ```PHP-DataType``` .

What do you think?

Thanks.

**Comment by Josantonius** [comment]
I think it's great that way. 

Thanks for your help :)
